### DISCUSSION TITLE
I have changed local ssh key recently, how could I generate proof

### DISCUSSION THREAD

**Discussion by yyjia** [FiloSottile/age#551] [Q&A] [answered]
Recently I changed my laptop so I have lost my old private key

**Comment by yyjia** [comment, marked as answer]
how could I  generate a proof of ownership of the Github account.

**Comment by cipriancraciun** [comment]
Based on other similar issues like yours that were received in the previous days, most likely your problem is with the `fluencelabs/dev-rewards` project as described here: <https://github.com/fluencelabs/dev-rewards/issues/19>

**Reply by yyjia** [reply]
yes stupid  decision
